### ISSUE TITLE
Filter while browsing

### ISSUE THREAD

**Issue by cassidyjames** [flathub/website#1910]
Right now it looks like filters like the FOSS filter only show up on the search results page. It would be nice to be able to browse using the filters as well, e.g. if you only want to explore FOSS apps.

**Comment by ssokolow** [comment]
Agreed. I hadn't even realized filtering had been implemented because my primary interaction with Flathub is to visit https://flathub.org/apps/collection/recently-added/ every so often to see if any more of the apps I use have been "made easier to custom-sandbox than Firejail" yet.

**Comment by cahfofpai** [comment]
Any progress on this issue?

**Comment by cahfofpai** [comment]
@cassidyjames Do you or somebody else plan to implement it, or would it be open for anybody to work on?

**Comment by razzeee** [comment]
I mostly see problems with this. Tthis might lead to empty pages/sections, without users understanding why that is.

**Comment by cahfofpai** [comment]
> I mostly see problems with this. Tthis might lead to empty pages/sections, without users understanding why that is.

An empty state could be implemented for cases like this, which informs the users about the global filter being applied leading to the empty page / section.

**Comment by cahfofpai** [comment]
Would it be relevant to take server load into consideration for this feature?
Because when using the global filter, a search query would be necessary for every page call to filter the apps to be displayed.
Would the additional load be manageable with the current server setup?

**Comment by razzeee** [comment]
We generate static pages on demand, so it shouldn't be too bad, but the more mutations there are, the slower will our page be.

**Comment by cahfofpai** [comment]
Would you generate static pages for all combinations of filter criteria?
Categories could be left out as it does not make sense as global filter. But if all of the other filter criteria would be allowed for the global filter, there would be nonetheless many combinations:
3 (License) * 3 (Verification) * 3 (App Type) * 3 (Arch) = 81 possible combinations
(for a criteria with 2 choices there are 3 options, as it is optional and thus could also be left blank)

**Comment by razzeee** [comment]
x number of translations
